.gitignore

```diff
@@ -1,3 +1,9 @@
+# Emacs/Vim turds
+.\#*\#
+\#*\#
+*~
+.tern-port
+
```

Comment:
These probably belong in your global gitignore

Reply:
Fair point, sorry